sdk/identity/azure-identity/samples/README.md

```diff
@@ -4,7 +4,7 @@ languages:
   - python
 products:
   - azure
-  - azure-identity
+  - azure-sdks
```

Comment:
JS uses `azure-active-directory` here. Would it make sense to use the same?

Reply:
Would that cause these samples to show up in Azure AD documentation?

Reply:
@witemple-msft do you know?

Reply:
Answered my own question: it does, in the [samples browser](https://docs.microsoft.com/en-us/samples/browse/?expanded=azure&products=azure-active-directory&languages=javascript) at least. At first glance I'm thinking our samples shouldn't be grouped with Azure Active Directory samples. This library uses Azure AD but is narrowly focused on authenticating Azure SDK clients. Our samples are poor demonstrations of Azure AD features and in general just noise for people not using the SDK.

Reply:
Where would someone expect to find the identity SDK samples in the microsoft docs?

Reply:
For example, in form recognizer, our MS docs menu has a drop down for "Samples" where our samples readme gets published: https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/overview?tabs=v2-1

Reply:
Authentication is covered in the [SDK conceptual docs](https://docs.microsoft.com/azure/developer/python/azure-sdk-authenticate). As for the sample browser, the taxonomy is ordered by product, so I think azure-identity samples belong under `azure` (though I think that's too broad a category to be really useful) and `azure-sdks`.